docs-v2/components/homepage/CoreValueCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { Heading, Icon, majorScale, Pane, Paragraph, useTheme } from "evergreen-ui"
+
+interface Props {
+  icon: typeof Icon,
+  title: string,
+  description: string
+}
+
+const ValueCard = ({icon, title, description}: Props) => {
+  const { colors } = useTheme()
```

Comment:
In order to get rid of the type error, you might want to do `const { colors } = useTheme() as any`